openjdk/playlist.xml

```diff
@@ -64,6 +120,31 @@
 			<impl>hotspot</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>hotspot_serviceability</testCaseName>
+		<variations>
+			<variation>Mode150</variation>
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) $(JVM_NATIVE_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS)$(Q) \
+	-w $(Q)$(REPORTDIR)$(D)work$(Q) \
+	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
+	-jdk:$(Q)$(TEST_JDK_HOME)$(Q) \
+	$(Q)$(JTREG_HOTSPOT_TEST_DIR): hotspot_serviceability$(Q); \
```

Comment:
need to remove the whitespace between ':' and 'hotspot_serviceability' (I see that I had actually added it in the example in the issue, but it won't work with the whitespace present)

Reply:
I removed the whitespace, I hope it will work now.